From 8a61babf5b1ac20ba0c63770a3e1bc714d8c55fd Mon Sep 17 00:00:00 2001
From: Chunjian zheng <chunjian.zheng@amlogic.com>
Date: Fri, 30 May 2014 18:26:53 +0800
Subject: [PATCH 4268/5965]  PD#83628:PMU1218: pmu driver update.

    1.modify charging target voltage and usb limit current;
    2,modify LDO7 voltage;
---
 arch/arm/boot/dts/amlogic/meson8_k150_v1.dtd    | 4 ++--
 arch/arm/boot/dts/amlogic/meson8_k160_v1.dtd    | 4 ++--
 arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd | 4 ++--
 arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd   | 4 ++--
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/meson8_k150_v1.dtd b/arch/arm/boot/dts/amlogic/meson8_k150_v1.dtd
index 93ae29a85964..6b1b7463197b 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k150_v1.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k150_v1.dtd
@@ -641,7 +641,7 @@ sdio{
         pmu_battery_cap = <3670>;           /** battery capacity                     */
         pmu_battery_technology = <4>;       /** POWER_SUPPLY_TECHNOLOGY_LiFe         */
         pmu_battery_name = "PTI PL336078";  /** battery name                         */
-        pmu_init_chgvol = <4200000>;        /** target charge voltage                */
+        pmu_init_chgvol = <4050000>;        /** target charge voltage                */
         pmu_init_chgend_rate = <10>;        /** set initial charing end current rate */
         pmu_init_chg_enabled = <1>;         /** set initial charing enabled          */
         pmu_init_adc_freq = <25>;           /** set initial adc frequency            */
@@ -653,7 +653,7 @@ sdio{
         pmu_resume_chgcur = <10>;           /** resume charge current, 0.1c          */
         pmu_shutdown_chgcur = <20>;         /** shutdown charge current, 0.2c        */
         pmu_usbcur_limit = <1>;             /** enable usb current limit?            */
-        pmu_usbcur = <500>;                 /** usb limit current value if enabled   */
+        pmu_usbcur = <1000>;                 /** usb limit current value if enabled   */
         pmu_usbvol_limit = <1>;             /** enable usb vbus voltage limit        */
         pmu_usbvol = <4400>;                /** voltage limit value if enabled       */
         pmu_pwroff_vol = <3200>;            /** hardware force power off voltage     */
diff --git a/arch/arm/boot/dts/amlogic/meson8_k160_v1.dtd b/arch/arm/boot/dts/amlogic/meson8_k160_v1.dtd
index c5b42eedcf12..bf7d72d13329 100755
--- a/arch/arm/boot/dts/amlogic/meson8_k160_v1.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8_k160_v1.dtd
@@ -641,7 +641,7 @@ sdio{
         pmu_battery_cap = <3670>;           /** battery capacity                     */
         pmu_battery_technology = <4>;       /** POWER_SUPPLY_TECHNOLOGY_LiFe         */
         pmu_battery_name = "PTI PL336078";  /** battery name                         */
-        pmu_init_chgvol = <4200000>;        /** target charge voltage                */
+        pmu_init_chgvol = <4050000>;        /** target charge voltage                */
         pmu_init_chgend_rate = <10>;        /** set initial charing end current rate */
         pmu_init_chg_enabled = <0>;         /** set initial charing enabled          */
         pmu_init_adc_freq = <25>;           /** set initial adc frequency            */
@@ -653,7 +653,7 @@ sdio{
         pmu_resume_chgcur = <10>;           /** resume charge current, 0.1c          */
         pmu_shutdown_chgcur = <20>;         /** shutdown charge current, 0.2c        */
         pmu_usbcur_limit = <1>;             /** enable usb current limit?            */
-        pmu_usbcur = <500>;                 /** usb limit current value if enabled   */
+        pmu_usbcur = <1000>;                 /** usb limit current value if enabled   */
         pmu_usbvol_limit = <1>;             /** enable usb vbus voltage limit        */
         pmu_usbvol = <4400>;                /** voltage limit value if enabled       */
         pmu_pwroff_vol = <3200>;            /** hardware force power off voltage     */
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd b/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
index add63e45db96..ff237a73b771 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m101_512M.dtd
@@ -788,7 +788,7 @@ void root_func(){
         pmu_battery_cap = <2834>;           /** battery capacity                     */
         pmu_battery_technology = <4>;       /** POWER_SUPPLY_TECHNOLOGY_LiFe         */
         pmu_battery_name = "PTI PL336078";  /** battery name                         */
-        pmu_init_chgvol = <4200000>;        /** target charge voltage                */
+        pmu_init_chgvol = <4050000>;        /** target charge voltage                */
         pmu_init_chgend_rate = <20>;        /** set initial charing end current rate */
         pmu_init_chg_enabled = <1>;         /** set initial charing enabled          */
         pmu_init_adc_freq = <25>;           /** set initial adc frequency            */
@@ -800,7 +800,7 @@ void root_func(){
         pmu_resume_chgcur = <20>;           /** resume charge current, 0.2c          */
         pmu_shutdown_chgcur = <30>;         /** shutdown charge current, 0.3c        */
         pmu_usbcur_limit = <1>;             /** enable usb current limit?            */
-        pmu_usbcur = <500>;                 /** usb limit current value if enabled   */
+        pmu_usbcur = <1000>;                 /** usb limit current value if enabled   */
         pmu_usbvol_limit = <1>;             /** enable usb vbus voltage limit        */
         pmu_usbvol = <4400>;                /** voltage limit value if enabled       */
         pmu_pwroff_vol = <3200>;            /** hardware force power off voltage     */
diff --git a/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd b/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
index 19bc97add837..63623ef4c66f 100755
--- a/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
+++ b/arch/arm/boot/dts/amlogic/meson8b_m102_1G.dtd
@@ -767,7 +767,7 @@ void root_func(){
         pmu_battery_cap = <3560>;           /** battery capacity                     */
         pmu_battery_technology = <4>;       /** POWER_SUPPLY_TECHNOLOGY_LiFe         */
         pmu_battery_name = "PTI PL336078";  /** battery name                         */
-        pmu_init_chgvol = <4200000>;        /** target charge voltage                */
+        pmu_init_chgvol = <4050000>;        /** target charge voltage                */
         pmu_init_chgend_rate = <20>;        /** set initial charing end current rate */
         pmu_init_chg_enabled = <1>;         /** set initial charing enabled          */
         pmu_init_adc_freq = <25>;           /** set initial adc frequency            */
@@ -779,7 +779,7 @@ void root_func(){
         pmu_resume_chgcur = <20>;           /** resume charge current, 0.2c          */
         pmu_shutdown_chgcur = <30>;         /** shutdown charge current, 0.3c        */
         pmu_usbcur_limit = <1>;             /** enable usb current limit?            */
-        pmu_usbcur = <500>;                 /** usb limit current value if enabled   */
+        pmu_usbcur = <1000>;                 /** usb limit current value if enabled   */
         pmu_usbvol_limit = <1>;             /** enable usb vbus voltage limit        */
         pmu_usbvol = <4400>;                /** voltage limit value if enabled       */
         pmu_pwroff_vol = <3200>;            /** hardware force power off voltage     */
-- 
2.19.0

